modules/mocked_authentication/app/services/mocked_authentication/mock_credential_info_creator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module MockedAuthentication
+  class MockCredentialInfoCreator < ApplicationController
+    attr_reader :credential_info
+
+    def initialize(credential_info:)
+      @credential_info = credential_info
+    end
+
+    def perform
+      validate_credential_info
+      create_mock_credential_info
+      @mock_credential_info
+    end
+
+    private
+
+    def validate_credential_info
+      raise 'Credential Info missing' unless @credential_info
```

Comment:
Since you have the `attr_reader` for `credential_info` you can remove the `@` all these places that you're reading the variable